.github/workflows/example-workflow.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+on:
+  push:
+    branches:
+      - example
```

Comment:
If we are not testing this on push to main, are we keep this in `.github/workflows` so that users who clone get an out of the box working example?

Reply:
Right, so they don't have to create their own repo to copy over the example. I'll add the clone step to the example instructions.